### ISSUE TITLE
SplitButton & ToggleSplitButton: Incorrect tab order

### ISSUE THREAD

**Issue by Athari** [irihitech/Semi.Avalonia#701]
When pressing the <kbd>Tab</kbd> key on a page with `SplitButton` or `ToggleSplitButton`, first the arrow button is focused and only then the main button.

I think the correct tab order should be the opposite of that, consistently from left to right.
